workflow/agents/features.mdx

```diff
@@ -106,6 +121,27 @@ const wikiTool = new WikipediaQueryRun({
 
 For available toolkits, you can explore the [LangChain Toolkits](https://js.langchain.com/v0.1/docs/modules/agents/tools/toolkits/) or other AI SDK-compatible toolkits like [Agentic](https://agentic.so/sdks/ai-sdk).
 
+By default, the Workflow SDK will wrap the execute/invoke methods of the tools you pass with [`context.run`](/workflow/basics/context#context-run) to run them as a Workflow step. This means, you can't use steps like context.call, context.notify etc in execute/invoke right away. If you want to define steps in invoke/execute, you can use the `executeAsStep` option:
```

Comment:
Instead of 
`you can use the "executeAsStep" option`, 
I think this one is less confusing: 
`you can set the "executeAsStep" option to false as follows:`

If I don't check the code the description feels like I need to set `executeAsStep` to true.